app/views/companies/form.php

```diff
@@ -1,25 +1,25 @@
 <panel class="form">
   <div class="content">
-    <headline><?php vecho('headline'); ?> Company Profile</headline>
-    <?php if (vget('submitname') == 'add') { ?>
+    <headline><?php View::echof('headline'); ?> Company Profile</headline>
+    <?php if (View::get('submitname') == 'add') { ?>
       <i>This form auto-saves, so you may return to finish the form later.</i>
     <?php } ?>
     <form method="post" id="company">
       <?php
-        if (vget('_id') !== null) {
-          $id = vget('_id');
-          echo ' &nbsp; ' . vlinkto('<input type="button" value="View Company Profile" /><br /><br />', 'company', array('id' => $id));
+        if (View::get('_id') !== null) {
+          $id = View::get('_id');
+          echo ' &nbsp; ' . View::linkto('<input type="button" value="View Company Profile" /><br /><br />', 'company', array('id' => $id));
```

Comment:
Even though PHP is case-insensitive, let's try to maintain all the same cases. (ie here is View::linkTo)